docs/reference/commandline/service_create.md

```diff
@@ -177,6 +177,45 @@ $ docker service create \
   redis:3.0.6
 ```
 
+### Container networking
+The containers provisioned in docker swarm mode can be accessed in service discovery either via a Virtual IP (VIP) and routed through the docker swarm ingress overlay network. Or via a DNS round robbin (DNSRR). This configuration is set using the `--endpoint-mode [vip|dnsrr]` setting. Note that dnsrr is incompatible with `--published` ports.
```

Comment:
This information is incorrect. VIP is not on the ingress overlay network. You need to create a user-defined overlay network in order to use VIP or DNSRR. See: https://github.com/docker/docker/pull/25420

We should not conflate ingress, which is ('--published' ports) with swarm-internal overlay networking. I am working on a second guide to help describe that.